### PR TITLE
SolvedQ15

### PR DESCRIPTION
**Pull request by AjaySunda**

### CODE REVIEW

File: 2. flow_control/Q15.c [hunk 1]
```diff
@@ -1 +1,26 @@
 // Write a program to read an array of n integers and print the array in reverse order using a for loop.
+#include<stdio.h>
+int main(){
+    int n,array[100],i;
```
Comment (AbhishekSunda) [inline 2. flow_control/Q15.c]: you can use this array[n]

File: 2. flow_control/Q15.c [hunk 2]
```diff
@@ -1 +1,26 @@
...
+    printf("Enter the number of elements in the array = ");
+    scanf("%d", &n);
+
+    for ( i = 0; i <= n-1; i++)
```
Comment (AbhishekSunda) [inline 2. flow_control/Q15.c]: second condition in for loop can be written like this i<n for brevity

File: 2. flow_control/Q15.c [hunk 3]
```diff
@@ -1 +1,26 @@
...
+    printf("Enter the number of elements in the array = ");
+    scanf("%d", &n);
+
+    for ( i = 0; i <= n-1; i++) 
+    {
+        printf("Enter integer value : ");
+        scanf("%d", &array[i]);
+    }
+    printf("array : ");
+    for ( i = 0; i <= n-1; i++)
```
Comment (AbhishekSunda) [inline 2. flow_control/Q15.c]: same here

**Review by AbhishekSunda — CHANGES_REQUESTED** [review]

File: 2. flow_control/Q15.c (hunk 2 above)
Comment (AjaySunda) [reply 2. flow_control/Q15.c]: printf("\nreverse array : ");
    for ( i = n-1; i >= 0; i--)
    {
        printf("%d ", array[i]);
    }
 What to use in place of n-1 in this situation?

Comment (AbhishekSunda) [reply 2. flow_control/Q15.c]: no need, its correct here.

**Review by AbhishekSunda — APPROVED** [review]

**Pull request merged**